lib/nostrum/api.ex

```diff
@@ -1255,12 +1255,9 @@ defmodule Nostrum.Api do
 
   ## Examples
 
-  ```Elixir
-  Nostrum.Api.modify_guild(81384788765712384, name: "Elixir Guild")
-  {:ok, %Nostrum.Struct.Channel{id: 81384788765712384, name: "Elixir Guild"}}
-
-  Nostrum.Api.modify_channel(81384788765712384)
-  {:ok, %Nostrum.Struct.Channel{id: 81384788765712384}}
+  ```elixir
+  iex> {:ok, %Nostrum.Struct.Guild{id: 451824027976073216, name: "Nose Drum"}} = Nostrum.Api.modify_guild(451824027976073216, name: "Nose Drum")
```

Comment:
We don't need the `iex>` here if we're not doing a doctest, IMO it just makes it look messy.

I'm fine with the example as is, or we could just have a typical `call -> response` example. Up to you.